scripts/functions/utility_system

```diff
@@ -30,6 +30,8 @@ __rvm_remote_extension()
 # params: ruby-string-to-transform
 __rvm_ruby_package_file()
 {
+  # What's this? Is it just for binary rubies? Do we need to change something for TruffleRuby?
+  # We already set rvm_ruby_package_file in __rvm_select_interpreter_truffleruby().
```

Comment:
What's this function?
Is it just for binary rubies?
Do we need to change something for TruffleRuby?
Note that we already set rvm_ruby_package_file in `__rvm_select_interpreter_truffleruby()`.

Reply:
This would make sense if we install TruffleRuby, prepare binary for it, upload it to RVM binaries and then people download them, I'm not sure this would work with TruffleRuby license, but still could be used for multiserver deploys in private environment.

I guess we can add this in separate PR.